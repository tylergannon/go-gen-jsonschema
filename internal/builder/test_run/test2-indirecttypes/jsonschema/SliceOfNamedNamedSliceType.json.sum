1326b643b3603f08