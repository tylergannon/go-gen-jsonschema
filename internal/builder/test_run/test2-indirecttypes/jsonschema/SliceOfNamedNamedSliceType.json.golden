{
  "type": "array",
  "description": "SliceOfNamedNamedSliceType is a slice of the named slice type.",
  "items": {
    "type": "array",
    "description": "NamedNamedSliceType is another level of indirection from NamedSliceType",
    "items": {
      "type": "integer",
      "description": "IntType is Foobarbax"
    }
  }
}
